src/js/components/useQuantityInput.ts

```diff
@@ -64,7 +64,18 @@ const useQuantityInput: Theme.QuantityInput.Function = (
             }
 
             if (event.key === ENTER_KEY) {
-              updateQuantity(qtyInputGroup, 1);
+              if (qtyInput.value === '0') {
+                const targetItem = qtyInput.closest('.cart__item');
```

Comment:
I guess we should introduce a new entry in the selector-map 🤔 
https://github.com/PrestaShop/hummingbird/blob/develop/src/js/constants/selectors-map.ts